src/artikler/andre-muligheter/JobblystPanel.tsx

```diff
@@ -1,59 +1,111 @@
 import * as React from "react";
+import styled from "styled-components";
```

Comment:
Hvis du importerer fra styled-components/macro så får man litt mer forståelig className i devtools.. (tror next-apper fikser dette for deg automatisk, men tydeligvis ikke cra-apper)

Reply:
Takk! Den var jeg ikke klar over.  ^^